packages/toolpad-app/src/components/EditableText.tsx

```diff
@@ -11,25 +18,42 @@ interface EditableTextProps {
   onBlur: (event: React.FocusEvent<HTMLInputElement>) => void;
   variant?: TypographyVariant;
   size?: 'small' | 'medium';
+  inputProps?: SxProps;
```

Comment:
I find this slightly misleading - it's named as `inputProps` but it can only control `sx` - would it make sense to name it `inputStyle` or `inputSx` instead? 🤔